src/components/SearchResults.jsx

```diff
@@ -0,0 +1,104 @@
+import React, { useState } from "react";
+
+import Bookings from "./Bookings";
+import CustomerProfile from "./CustomerProfile";
+import TableRow from "./TableRow";
+function SearchResults({ results }) {
+  const [customerId, setCustomerId] = useState(null);
+  const sortedResults = [...results];
+  const [sortedField, setSortedField] = useState(null);
+  if (sortedField !== null) {
+    sortedResults.sort((a, b) => {
+      if (a[sortedField] < b[sortedField]) {
```

Comment:
`sort` will default to sorting by ASCII code so you may want to ensure you lower case the items in the array before comparing them.

Reply:
Sort was difficult to implement I know its still not working as it should be.